bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/MainCommand.java

```diff
@@ -74,6 +74,7 @@ public MainCommand(String name) {
         new CmdStats(this, "stats");
         new CmdStart(this, "forceStart");
         new CmdStart(this, "start");
+        new CmdPlayAgain(this, "playagain");
```

Comment:
If you want to add the play-again feature you must think and implement it with BedWarsProxy. Updates must be equal and available for every supported server type.